cxx/isce3/cuda/focus/Backproject.h

```diff
@@ -85,6 +90,8 @@ backproject(std::complex<float>* out,
             DryTroposphereModel dry_tropo_model = DryTroposphereModel::TSX,
             const Rdr2GeoBracketParams& rdr2geo_params = {},
             const Geo2RdrBracketParams& geo2rdr_params = {},
-            int batch = 1024, float* height = nullptr);
+            int batch = 1024,
+            const std::optional<isce3::core::ChebyKernel<float>> window = std::nullopt,
```

Comment:
As mentioned in a previous comment, let's pass this by const ref.

```suggestion
            const std::optional<isce3::core::ChebyKernel<float>>& window = std::nullopt,
```